ethcontract/Cargo.toml

```diff
@@ -17,20 +17,23 @@ to Ethereum smart contracts.
 name = "ethcontract"
 
 [features]
+aws-kms = ["aws-config", "aws-sdk-kms", "rlp"]
 default = ["derive", "http-tls", "ws-tls-tokio", "derive-http"]
 derive = ["ethcontract-derive"]
 derive-http = ["ethcontract-derive/http"]
 http = ["web3/http"]
-http-tls = ["http", "web3/http-tls"]
 http-native-tls = ["http", "web3/http-native-tls"]
 http-rustls-tls = ["http", "web3/http-rustls-tls"]
-ws-tokio = ["web3/ws-tokio"]
+http-tls = ["http", "web3/http-tls"]
+ipc-tokio = ["web3/ipc-tokio"]
 ws-async-std = ["web3/ws-async-std"]
-ws-tls-tokio = ["web3/ws-tls-tokio"]
 ws-tls-async-std = ["web3/ws-tls-async-std"]
-ipc-tokio = ["web3/ipc-tokio"]
+ws-tls-tokio = ["web3/ws-tls-tokio"]
+ws-tokio = ["web3/ws-tokio"]
 
 [dependencies]
+aws-config = { version = "0.55", optional = true }
+aws-sdk-kms = { version = "0.28", optional = true }
```

Comment:
I want to point out again that all of the aws rust crates have this disclaimer in their [readme](https://github.com/awslabs/smithy-rs/blob/2987cbdd9104c7350f2c26b10673b9af180264f8/aws/rust-runtime/aws-config/README.md):

> **Please Note: The SDK is currently released as an alpha and is intended strictly for feedback purposes only. Do not use this SDK for production workloads.**

Yet we are planning on using this in a production workload. This doesn't have to block us but I don't want it to be overlooked.

Reply:
Yeah didn't mean this as a blocker. I just wanted to note it and then didn't have time to finish the review.